nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -927,6 +930,29 @@ class ReconAllInputSpec(CommandLineInputSpec):
     )
     flags = InputMultiPath(traits.Str, argstr="%s", desc="additional parameters")
 
+    # Longitudinal runs
```

Comment:
We might need to introduce additional constraints to avoid using cross-sectional specific arguments with base and long modes.

I am thinking of the `T1_files`, `T2_file` and `FLAIR_file` parameters at least. These should probably get a requirement on `subject_id`, which would achieve what we want thanks to your `xor` constraints.

Reply:
That didn't cross my mind but excellent thought. I've add some requires and will test some variations next week to make sure that works as expected.

Reply:
Looks good to me :+1: